package-lock.json

```diff
@@ -9,7 +9,8 @@
         "packages/*"
       ],
       "dependencies": {
-        "@motion-canvas/core": "^3.0.0"
+        "@motion-canvas/core": "^3.0.0",
+        "glob": "^9.3.2"
```

Comment:
You should revert the whole `package-lock.json` file, remove `node_modules`, and perform the installation again because the file seems corrupted

Reply:
Done!